functions/go/gatekeeper/go.mod

```diff
@@ -1,12 +1,88 @@
 module github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/gatekeeper
 
-go 1.16
+go 1.17
 
 require (
-	github.com/open-policy-agent/frameworks/constraint v0.0.0-20210522003146-5c034948ac29
-	github.com/open-policy-agent/gatekeeper v0.0.0-20210527161344-e229247f04d1 // This is v3.5.1. It has a semver major version of 2 or higher and is not a Go module yet.
-	github.com/spf13/cobra v1.1.1
-	k8s.io/apimachinery v0.20.2
+	github.com/open-policy-agent/frameworks/constraint v0.0.0-20211025234246-f478d8a555b5
+	github.com/open-policy-agent/gatekeeper v0.0.0-20211115204247-3ba8e93049b5 // This is v3.7.0. It has a semver major version of 2 or higher and is not a Go module yet.
```

Comment:
The import path should be  `github.com/open-policy-agent/gatekeeper/v3` if it's compatible with go modules. Otherwise, we have to use SHA. It's not a big deal.

Reply:
gotcha!  Thanks